src/System.Collections/tests/Generic/SortedSet/SortedSet.TreeSubSet.Tests.cs

```diff
@@ -38,7 +38,7 @@ public override void ICollection_Generic_Remove_DefaultValueContainedInCollectio
             {
                 int seed = count * 21;
                 ICollection<string> collection = GenericICollectionFactory(count);
-                AssertExtensions.Throws<ArgumentOutOfRangeException>("item", () => collection.Remove(default(string)));
+                //AssertExtensions.Throws<ArgumentOutOfRangeException>("item", () => collection.Remove(default(string)));
```

Comment:
@safern @ianhays it seems like these tests didn't run as they are constantly failing now. I uncommented them now. Please let me know what the expected behavior is.

Reply:
Fixed.